lbr_examples/scripts/lbr_joint_state_publisher.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python3
```

Comment:
this scripts seems redundant as joint states are already published via launching the joint state publisher